From 01ea0ddd202eb78a5d9336a83234cf47750a89e2 Mon Sep 17 00:00:00 2001
From: Anson Huang <b20788@freescale.com>
Date: Fri, 23 Dec 2011 10:11:00 +0800
Subject: [PATCH 0786/2463] ENGR00170891 [MX6]Disable WAIT mode and DVFS

WAIT mode and DVFS still have some defects, we need
to disable it by default until we make them works.

Signed-off-by: Anson Huang <b20788@freescale.com>
---
 arch/arm/mach-mx6/cpu.c       |    2 +-
 arch/arm/plat-mxc/dvfs_core.c |    5 -----
 2 files changed, 1 insertions(+), 6 deletions(-)

diff --git a/arch/arm/mach-mx6/cpu.c b/arch/arm/mach-mx6/cpu.c
index 81f691c..3b7a344 100644
--- a/arch/arm/mach-mx6/cpu.c
+++ b/arch/arm/mach-mx6/cpu.c
@@ -38,7 +38,7 @@ extern void mx6_wait(void);
 
 
 struct cpu_op *(*get_cpu_op)(int *op);
-bool enable_wait_mode = true;
+bool enable_wait_mode;
 u32 arm_max_freq = CPU_AT_1GHz;
 
 void __iomem *gpc_base;
diff --git a/arch/arm/plat-mxc/dvfs_core.c b/arch/arm/plat-mxc/dvfs_core.c
index 6616831..cacd3cc 100755
--- a/arch/arm/plat-mxc/dvfs_core.c
+++ b/arch/arm/plat-mxc/dvfs_core.c
@@ -1081,11 +1081,6 @@ static int __init dvfs_init(void)
 	dvfs_core_is_active = 0;
 	printk(KERN_INFO "DVFS driver module loaded\n");
 
-	/* Enable DVFS by default. */
-	if (start_dvfs() != 0)
-		printk(KERN_ERR "Failed to start DVFS\n");
-	printk(KERN_INFO "DVFS driver Enabled\n");
-
 	return 0;
 }
 
-- 
1.7.7.4

